src/main/java/persistence/sql/ddl/ColumnMetaInfoFactory.java

```diff
@@ -0,0 +1,37 @@
+package persistence.sql.ddl;
+
+import persistence.sql.ddl.annotation.AnnotationInfo;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ColumnMetaInfoFactory {
+
+    public static List<ColumnMetaInfo> createColumnMetaInfo(Field field) {
+        List<ColumnMetaInfo> result = new ArrayList<>();
+
+        for (Annotation annotation : field.getAnnotations()) {
+            AnnotationInfo annotationInfo = getAnnotationInfo(field, annotation);
+            result.addAll(annotationInfo.getColumnMetaInfos());
+        }
+
+        return result.stream().sorted(ColumnMetaInfo::compareTo).collect(Collectors.toList());
```

Comment:
```suggestion
        return result.stream()
        .sorted(ColumnMetaInfo::compareTo)
        .collect(Collectors.toList());
```

가독성과 디버깅의 장점이 있어서 저는 한줄에 점 하나만 표현하도록 작성하는 편입니다 😄